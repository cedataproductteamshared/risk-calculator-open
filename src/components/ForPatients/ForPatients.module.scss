.for-patients-container {
  padding-top: 3rem;
  padding-bottom: 3rem;
}

h4 {
  margin-bottom: 1.5rem;
}
ol {
  margin-top: 0.7rem;
}

li {
  margin-bottom: 0.5rem;
}
