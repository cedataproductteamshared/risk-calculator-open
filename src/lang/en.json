{
  "calcform.options.sex.male": "Male",
  "calcform.options.sex.female": "Female",
  "calcform.options.residence.north_america": "North America",
  "calcform.options.treatments.statin.high_intensity.label": "High Intensity",
  "calcform.options.treatments.statin.high_intensity.description": "Atorva â‰¥ 40mg; Rosuva â‰¥ 20 mg",
  "calcform.options.treatments.statin.moderate_intensity.label": "Moderate Intensityg",
  "calcform.options.treatments.statin.moderate_intensity.description": "Atorva 10-20 mg; Rosuva 5-10 mg; Prava 40-80 mg; Lova 40 mg; Fluva 80 mg daily dose; Simva 20-40 mg; Pitava 2-4 mg",
  "calcform.options.treatments.statin.low_intensity.label": "Low Intensity",
  "calcform.options.treatments.statin.low_intensity.description": "Any other dose",
  "calcform.options.treatments.p2y12.clopidogrel": "Clopidogrel",
  "calcform.options.treatments.p2y12.prasugrel": "Prasugrel",
  "calcform.options.treatments.p2y12.ticagrelor": "Ticagrelor",

  "calcform.validation.general.required": "This field is required",
  "calcform.validation.general.invalid_range": "Value must be between {min} and {max} {unit}",
  "calcform.validation.general.diabetes_medications_no_diabetes": "Calculator not valid for diabetes medication without diabetes diagnosis.",
  "calcform.validation.rivaroxaban.mixed_with_antiplatelets": "Calculator not valid for rivaroxaban with other non-aspirin anti-platelets",

  "calcform.patient_details_summary.cad": "CAD",
  "calcform.patient_details_summary.chf": "CHF",
  "calcform.patient_details_summary.cvd": "CVD",
  "calcform.patient_details_summary.pad": "PAD",
  "calcform.patient_details_summary.age": "Age {age}",
  "calcform.patient_details_summary.gender": "{gender, select, male {M} female {F} other {U}}",
  "calcform.patient_details_summary.smoker": "{isSmoker, select, true {Smoker} other {Non-Smoker}} {comm_appendix}",
  "calcform.patient_details_summary.sbp": "SBP {sbp} mmHG",
  "calcform.patient_details_summary.creatinine": "Cr {cr} mg/dl",
  "calcform.patient_details_summary.totalCl": "Total Cholesterol {totalCl} mg/dl",
  "calcform.patient_details_summary.ldlCl": "LDL-C {ldlCl} mg/dl",
  "calcform.patient_details_summary.diabetes": "{diabetes, select, true {Diabetes} other {}}",
  "calcform.patient_details_summary.no_details": "No patient details",

  "calcform.patient_treatments_summary.statin": "Statin ({intensity} {medication})",
  "calcform.patient_treatments_summary.pcsk9": "PCSK9 Inhibitors",
  "calcform.patient_treatments_summary.ezetimibe": "Ezetimibe",
  "calcform.patient_treatments_summary.sglt2": "SGLT2 Inhibitors",
  "calcform.patient_treatments_summary.glp1": "GLP1 Receptor Agonists",
  "calcform.patient_treatments_summary.aspirin": "Aspirin",
  "calcform.patient_treatments_summary.p2y12": "P2Y12Inhibitors ({p2y12})",
  "calcform.patient_treatments_summary.rivaroxaban": "Rivaroxaban",
  "calcform.patient_treatments_summary.no_details": "No current treatment",

  "calcform.footer.button.calculate": "Calculate Risk Score",
  "calcform.footer.button.restore": "Restore",

  "calcform.form.info.provider_to_confirm": "Provider to confirm.",
  "calcform.form.info.calculator_validity": "Calculator only valid with one or more locations of cardiovascular disease.",
  "calcform.form.section_header.patient_details": "Patient details",
  "calcform.form.section_header.current_treatments": "Current treatments",
  "calcform.form.section_subheader.demographics": "Demographics and Lab Results",
  "calcform.form.section_subheader.comorbidities": "Comorbidities",
  "calcform.form.section_subheader.lipid_lowering": "Lipid-lowering",
  "calcform.form.section_subheader.antiplatelet": "AntiPlatelet / Anticoagulant",

  "calcform.form.control.age.label": "Age (years)",
  "calcform.form.control.age.placeholder": "45 - 89 years",
  "calcform.form.control.sex.label": "Sex",
  "calcform.form.control.residence.label": "Residence",
  "calcform.form.control.sbp.label": "Systolic Blood Pressure (mmHg)",
  "calcform.form.control.sbp.placeholder": "100 - 200 mmHg",
  "calcform.form.control.creatinine.label": "Creatinine (mg/dL)",
  "calcform.form.control.creatinine.placeholder": "0.3 - 6.0 mg/dL",
  "calcform.form.control.totalCholesterol.label": "Total Cholesterol (mg/dL)",
  "calcform.form.control.totalCholesterol.placeholder": "100 - 400 mg/dL",
  "calcform.form.control.ldlCholesterol.label": "LDL-Cholesterol (mg/dL)",
  "calcform.form.control.ldlCholesterol.placeholder": "0 - 400 mg/dL",
  "calcform.form.control.cad.label": "Coronary Artery Disease",
  "calcform.form.control.cvd.label": "Cerebrovascular Disease",
  "calcform.form.control.pad.label": "Peripheral Artery Disease",
  "calcform.form.control.atrial_fibrillation.label": "Atrial Fibrillation",
  "calcform.form.control.chf.label": "Congestive Heart Failure",
  "calcform.form.control.diabetes.label": "Diabetes",
  "calcform.form.control.smoking.label": "Current Smoking",
  "calcform.form.control.statin.label": "Statin*",
  "calcform.form.control.statin.warning": "Unable to determine intensity by given statin value ({medication}). Please select one manually.",
  "calcform.form.control.pcsk9.label": "PCSK9 Inhibitor (Alirocumab, Evolocumab)",
  "calcform.form.control.ezetimibe.label": "Ezetimibe",
  "calcform.form.control.sglt2.label": "SGLT2 Inhibitor (e.g. Dapagliflozin, Empagliflozin, etc.)",
  "calcform.form.control.glp1.label": "GLP1 Receptor Agonists (e.g. Semaglutide, Liraglutide, etc.)",
  "calcform.form.control.aspirin.label": "Aspirin",
  "calcform.form.control.p2i12.label": "P2Y12 Inhibitors",
  "calcform.form.control.rivaroxaban.label": "Low-dose Rivaroxaban (2.5 mg)",

  "treatments.form.info.select_treatment": "Select intended treatments to calculate updated estimated risk.",
  "treatments.form.section_header.intended_treatments": "Intended treatments",
  "treatments.form.section_subheader.lipid_lowering": "Lipid-lowering",
  "treatments.form.section_subheader.antiplatelet": "AntiPlatelet / Anticoagulant",
  "treatments.form.section_subheader.diabetes": "Diabetes",

  "treatments.form.control.statin.label": "Statin*",
  "treatments.form.control.pcsk9.label": "PCSK9 Inhibitor (Alirocumab, Evolocumab)",
  "treatments.form.control.ezetimibe.label": "Ezetimibe",
  "treatments.form.control.sglt2.label": "SGLT2 Inhibitor (e.g. Dapagliflozin, Empagliflozin, etc.)",
  "treatments.form.control.glp1.label": "GLP1 Receptor Agonists (e.g. Semaglutide, Liraglutide, etc.)",
  "treatments.form.control.aspirin.label": "Aspirin",
  "treatments.form.control.p2i12.label": "P2Y12 Inhibitors",
  "treatments.form.control.rivaroxaban.label": "Low-dose Rivaroxaban (2.5 mg)",



  "graph.data.label.no_treatment": "Without treatment*",
  "graph.data.label.current_treatment": "With current treatment",
  "graph.data.label.intended_treatment": "With intended treatment",

  "navbar.tabOne": "Risk Score",
  "navbar.tabTwo": "For Patients",
  "navbar.tabThree": "For Providers",

  "patientBanner.details": "{age} years, {gender}, {month}/{day}/{year}",
  "patientBanner.appTitle": "ASCVD Secondary Prevention Risk Calculator",
  "patientBanner.appSubtitle": "10-Year Risk Calculator for Established ASCVD (SMART-REACH)",
  "patientBanner.male": "Male",
  "patientBanner.female": "Female",

  "footer.footnote.statin": "The calculator is not intended to show how risk increases or changes if statin therapy were stopped or was not initiated",
  "footer.note.title": "Please Note",
  "footer.note.text": "This tool is meant to provide estimated 10-year risks and simulate changes on therapy.  Decisions regarding therapy should be made by providers.  Therapy options are displayed for simulation purposes and do not represent therapy recommendations.  Patient eligibility for therapy should be determined by their doctor/provider.  Risk estimates on current therapy and intended therapy based on clinical-trial based relative risk reductions.  See \"For Providers\" Tab for more information.",
  "footer.bp.title": "Blood Pressure",
  "footer.bp.text": "Blood pressures are used to calculate 10-year risk. Relative risk reductions for blood pressure lowering depend on starting blood pressure and the patient population. Current guideline-recommended BP goals for patients with established cardiovascular disease are <130/<80.",
  "footer.smoking.title": "Smoking",
  "footer.smoking.text": "Smoking status used to calculate 10-year risk. Quitting smoking will lower risk. The impact of smoking cessation on 10-year risk depends on duration since quitting.",
  "footer.about.title": "About the Score",
  "footer.about.text": "The SMART-REACH score provides 10-year risk of cardiovascular events in patients with established cardiovascular disease. See provider tab for more details. Kaasenbrood L et al. J Am Heart Assoc 2018;7:e009217",

  "message.simple": "A simple message.",
  "message.argument": "Hi, {name}! ðŸ‘‹",
  "message.plural": "{count, plural, one {# item} other {# items}}",
  "message.select": "{gender, select, male {Mr} female {Mrs} other {User}}",
  "message.text-format": "Hi, <b>John</b>!",
  "message.number-format": "Formatted number: {num, number, ::K}",
  "message.currency-format": "Formatted currency: {amount, number, ::currency/USD}"
}
