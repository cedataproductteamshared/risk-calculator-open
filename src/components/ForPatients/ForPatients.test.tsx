import React from 'react';
import { render } from '@testing-library/react';
import {IntlProvider} from "react-intl";
import en from "../../lang/en.json";
import ForPatients from "./ForPatients";

test('Succesful rendering', () => {
  const element = render(
      <IntlProvider locale="en" messages={en}>
        <ForPatients/>
      </IntlProvider>
  );
  expect(element).toMatchSnapshot();
});
