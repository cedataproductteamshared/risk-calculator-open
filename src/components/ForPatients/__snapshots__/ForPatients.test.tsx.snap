// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Succesful rendering 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="for-patients-container container"
      >
        <h4>
          What does the calculator estimate?
        </h4>
        <p>
          The calculator provides an estimate of your risk of heart attack, stroke, or death from cardiovascular disease in the next 10 years. This calculator is based on a largestudy of over 40,000 people with cardiovascular disease. Your risk is only an estimate – other factors about your health, your diet and exercise patterns, and your treatments may raise or lower your risk. You may already be on some therapies that lower your risk of heart disease. In this case, the calculator shows your risk without any treatments, then shows how your risk is already being lowered by the therapies you are taking.
        </p>
        <h4>
          How do I interpret this number?
        </h4>
        <p>
          If your risk score is 20%, the calculator estimates that 20 out of 100 people like you will have a heart attack, stroke, or cardiovascular death in the next 10 years.
        </p>
        <h4>
          What can I do to lower my risk of stroke, heart attack, or cardiovascular death?
        </h4>
        <p>
          The calculator already shows you some things you can do to lower your risk. You may or may not be eligible for these therapies- some of these should not be used in combination with other medications or in people with certain medical conditions. Make sure to talk to your doctor about what treatments you are eligible for.
        </p>
        These are explained below:
        <br />
        <ol>
          <li>
            <b>
              Control your LDL cholesterol:
            </b>
             For people with cardiovascular disease, the guidelines recommend a LDL cholesterol of 70 mg/dL or lower. High-intensity statins are the most effective way to do this and lower your risk of heart disease. Two statins: rosuvastatin (20 mg or higher) and atorvastatin (40 mg or higher) are considered “high intensity” and are recommended by guidelines for people with cardiovascular disease.
            <br />
            <br />
            When someone cannot tolerate a statin or who has LDL over 70 on a statin, ezetimibe can be added to statin therapy. Ezetimibe is a non-statin that lowers LDL cholesterol and is very well tolerated. After ezetimibe, injectable medications called PCSK9 inhibitors can be used to further lower LDL cholesterol and lower risk of heart disease.
          </li>
          <li>
            <b>
              Take an antiplatelet medication:
            </b>
             All people with cardiovascular disease should be on at least one medication to block platelets and help prevent heart attacks and strokes. Talk to your doctor about the best option for you- in many people this can be 81 mg of Aspirin. Others, for example those with recent stents in heart arteries, recent strokes or heart attacks, or peripheral artery disease may lower their risk further with a different anti-platelet medication such as clopidogrel (brand name Plavix), ticagrelor (brand name Brillinta), or prasugrel (brand name Effient).
          </li>
          <li>
            <b>
              Consider a blood thinner:
            </b>
             In addition to medications to block platelets, some patients may benefit from a low dose blood thinner to further help prevent clots called rivaroxaban (2.5 mg). This includes people with peripheral arterial disease and others who are at high-risk of heart attack. The dose is approved by the FDA. Because it is a blood thinner, people who take this medication are at higher risk of bleeding. Your doctor can help you evaluate your risk of bleeding and whether you are eligible for this therapy.
          </li>
          <li>
            <b>
              If you have type 2 diabetes, start a new medication
            </b>
             (SGLT2 inhibitor or GLP1 receptor agonist): Two different classes of medications are available that help lower the risk of cardiovascular events in people with type 2 diabetes. This includes SGLT2 Inhibitors and GLP1 receptor agonists. These are not insulin therapies and can help lower your hemoglobin A1c in addition to giving you extra protection against heart disease. Within the classes of medications, not all have the same benefit or safety profile. Some are given as weekly or daily injections while others are in pill forms. Ask your doctor about what may be a good choice for you.
          </li>
        </ol>
        <h4>
          What else can I do to lower my risk of heart disease?
        </h4>
        <ol>
          <li>
            <b>
              Control your blood pressure!
            </b>
             Blood pressure control is one of the most important ways to lower your risk of stroke, heart attack, and other diseases such as kidney disease and heart failure. Current guidelines recommend a blood pressure goal of &lt;130 (top number) and &lt;80 (bottom number) for most people with cardiovascular disease.
          </li>
          <li>
            <b>
              Don’t smoke!
            </b>
             It’s never too late to quit smoking. If you smoke, medication therapies are available to help you quit. Other ways to increase your success include nicotine replacement therapy (patches, gum, lozenges). Call 1-800-QUIT-NOW for more information.
          </li>
          <li>
            <b>
              Exercise!
            </b>
             Current guidelines recommend at least 75 minutes of high intensity exercise (like running or aerobics) or 150 minutes of moderate intensity exercise (like a bike ride or power walk) per week. One way to know if you’re doing moderate intensity exercise is how you feel- you should be able to speak but too winded to sing. If you are able to speak but only in short bursts, you’re probably doing high intensity exercise.
          </li>
          <li>
            <b>
              Eat well!
            </b>
             Avoid trans fats, saturated fats, and added sugars. Focus on whole grains, vegetables, and lean meats, especially fish. A Mediterranean diet withlean meats, fish, olive oil, nuts, fruits, and vegetables has been shown to lower the risk of heart disease.
          </li>
          <li>
            <b>
              Maintain a healthy weight!
            </b>
             Overweight and obese individuals are at increased risk of cardiovascular events and other complications like diabetes. Talk to your doctor about healthy ways to lose weight.
          </li>
        </ol>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="for-patients-container container"
    >
      <h4>
        What does the calculator estimate?
      </h4>
      <p>
        The calculator provides an estimate of your risk of heart attack, stroke, or death from cardiovascular disease in the next 10 years. This calculator is based on a largestudy of over 40,000 people with cardiovascular disease. Your risk is only an estimate – other factors about your health, your diet and exercise patterns, and your treatments may raise or lower your risk. You may already be on some therapies that lower your risk of heart disease. In this case, the calculator shows your risk without any treatments, then shows how your risk is already being lowered by the therapies you are taking.
      </p>
      <h4>
        How do I interpret this number?
      </h4>
      <p>
        If your risk score is 20%, the calculator estimates that 20 out of 100 people like you will have a heart attack, stroke, or cardiovascular death in the next 10 years.
      </p>
      <h4>
        What can I do to lower my risk of stroke, heart attack, or cardiovascular death?
      </h4>
      <p>
        The calculator already shows you some things you can do to lower your risk. You may or may not be eligible for these therapies- some of these should not be used in combination with other medications or in people with certain medical conditions. Make sure to talk to your doctor about what treatments you are eligible for.
      </p>
      These are explained below:
      <br />
      <ol>
        <li>
          <b>
            Control your LDL cholesterol:
          </b>
           For people with cardiovascular disease, the guidelines recommend a LDL cholesterol of 70 mg/dL or lower. High-intensity statins are the most effective way to do this and lower your risk of heart disease. Two statins: rosuvastatin (20 mg or higher) and atorvastatin (40 mg or higher) are considered “high intensity” and are recommended by guidelines for people with cardiovascular disease.
          <br />
          <br />
          When someone cannot tolerate a statin or who has LDL over 70 on a statin, ezetimibe can be added to statin therapy. Ezetimibe is a non-statin that lowers LDL cholesterol and is very well tolerated. After ezetimibe, injectable medications called PCSK9 inhibitors can be used to further lower LDL cholesterol and lower risk of heart disease.
        </li>
        <li>
          <b>
            Take an antiplatelet medication:
          </b>
           All people with cardiovascular disease should be on at least one medication to block platelets and help prevent heart attacks and strokes. Talk to your doctor about the best option for you- in many people this can be 81 mg of Aspirin. Others, for example those with recent stents in heart arteries, recent strokes or heart attacks, or peripheral artery disease may lower their risk further with a different anti-platelet medication such as clopidogrel (brand name Plavix), ticagrelor (brand name Brillinta), or prasugrel (brand name Effient).
        </li>
        <li>
          <b>
            Consider a blood thinner:
          </b>
           In addition to medications to block platelets, some patients may benefit from a low dose blood thinner to further help prevent clots called rivaroxaban (2.5 mg). This includes people with peripheral arterial disease and others who are at high-risk of heart attack. The dose is approved by the FDA. Because it is a blood thinner, people who take this medication are at higher risk of bleeding. Your doctor can help you evaluate your risk of bleeding and whether you are eligible for this therapy.
        </li>
        <li>
          <b>
            If you have type 2 diabetes, start a new medication
          </b>
           (SGLT2 inhibitor or GLP1 receptor agonist): Two different classes of medications are available that help lower the risk of cardiovascular events in people with type 2 diabetes. This includes SGLT2 Inhibitors and GLP1 receptor agonists. These are not insulin therapies and can help lower your hemoglobin A1c in addition to giving you extra protection against heart disease. Within the classes of medications, not all have the same benefit or safety profile. Some are given as weekly or daily injections while others are in pill forms. Ask your doctor about what may be a good choice for you.
        </li>
      </ol>
      <h4>
        What else can I do to lower my risk of heart disease?
      </h4>
      <ol>
        <li>
          <b>
            Control your blood pressure!
          </b>
           Blood pressure control is one of the most important ways to lower your risk of stroke, heart attack, and other diseases such as kidney disease and heart failure. Current guidelines recommend a blood pressure goal of &lt;130 (top number) and &lt;80 (bottom number) for most people with cardiovascular disease.
        </li>
        <li>
          <b>
            Don’t smoke!
          </b>
           It’s never too late to quit smoking. If you smoke, medication therapies are available to help you quit. Other ways to increase your success include nicotine replacement therapy (patches, gum, lozenges). Call 1-800-QUIT-NOW for more information.
        </li>
        <li>
          <b>
            Exercise!
          </b>
           Current guidelines recommend at least 75 minutes of high intensity exercise (like running or aerobics) or 150 minutes of moderate intensity exercise (like a bike ride or power walk) per week. One way to know if you’re doing moderate intensity exercise is how you feel- you should be able to speak but too winded to sing. If you are able to speak but only in short bursts, you’re probably doing high intensity exercise.
        </li>
        <li>
          <b>
            Eat well!
          </b>
           Avoid trans fats, saturated fats, and added sugars. Focus on whole grains, vegetables, and lean meats, especially fish. A Mediterranean diet withlean meats, fish, olive oil, nuts, fruits, and vegetables has been shown to lower the risk of heart disease.
        </li>
        <li>
          <b>
            Maintain a healthy weight!
          </b>
           Overweight and obese individuals are at increased risk of cardiovascular events and other complications like diabetes. Talk to your doctor about healthy ways to lose weight.
        </li>
      </ol>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
